docs/faq.md

```diff
@@ -197,4 +197,4 @@ Notice that just setting the background in the CSS does not have the desired eff
 [session-storage]: https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage
 [indexed-db]: https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API
 [browser-window]: api/browser-window.md
-[subpixel rendering example]: images/code_coverage_infra_diagram.png
\ No newline at end of file
+[subpixel rendering example]: images/code_coverage_infra_diagram.png
```

Comment:
I know this is an unrelated change, but this file is being changed automatically by the linter due to ` No newline at end of file`.